go.mod

```diff
@@ -114,4 +113,22 @@ require (
 	gopkg.in/validator.v2 v2.0.0-20210331031555-b37d688a7fb0
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	modernc.org/sqlite v1.13.1
+)
+
+require (
+	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
+	github.com/remyoudompheng/bigfft v0.0.0-20200410134404-eec4a21b6bb0 // indirect
+	golang.org/x/mod v0.4.2 // indirect
+	golang.org/x/tools v0.1.5 // indirect
+	golang.org/x/xerrors v0.0.0-20200804184101-5ec99f83aff1 // indirect
+	lukechampine.com/uint128 v1.1.1 // indirect
+	modernc.org/cc/v3 v3.34.0 // indirect
+	modernc.org/ccgo/v3 v3.11.2 // indirect
```

Comment:
Our production builds (Linux) don't use `cgo`. Am I right that this is some sort of alternative compiler which actually doesn't use `cgo`.

Reply:
That's right, you can read more about the project here: https://pkg.go.dev/modernc.org/sqlite

It's likely not as stable yet as the more popular https://github.com/mattn/go-sqlite3, which is why I didn't use it by default for Temporalite if cgo is enabled.